File: src/entities/Categorie.java
package entities;

/**
 *
 * @author Skander
 */
public class Categorie {
    private int id_categorie;
    private String categorieVoiture;

    public Categorie() {
    }

    public Categorie(int id_categorie ) {
                this.id_categorie = id_categorie;
        this.categorieVoiture = categorieVoiture;
        
    }

    public Categorie(String categorieVoiture) {
        this.categorieVoiture = categorieVoiture;
    }

   
    
    

    public int getIdcategorie() {
        return id_categorie;
    }

    public void setIdcategorie(int id_categorie) {
        this.id_categorie = id_categorie;
    }

    public String getCategorieVoiture() {
        return categorieVoiture;
    }

    public void setCategorieVoiture(String categorieVoiture) {
        this.categorieVoiture = categorieVoiture;
    }

    

    @Override

    
    public String toString() {
        return "Categorie{" + "id_categorie=" + id_categorie + ", categorieVoiture=" + categorieVoiture +  '}';
    }

    
   

   

   
    
    
    
    
    
}

File: src/Entity/Categorie.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Categorie
 *
 * @ORM\Table(name="categorie")
 * @ORM\Entity
 */
class Categorie
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_categorie", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idCategorie;

    /**
     * @var string
     *
     * @ORM\Column(name="categorieVoiture", type="string", length=30, nullable=false)
     */
    private $categorievoiture;

    /**
     * Get the value of idCategorie
     *
     * @return  int
     */ 
    public function getIdCategorie()
    {
        return $this->idCategorie;
    }

    /**
     * Set the value of idCategorie
     *
     * @param  int  $idCategorie
     *
     * @return  self
     */ 
    public function setIdCategorie(int $idCategorie)
    {
        $this->idCategorie = $idCategorie;

        return $this;
    }

    /**
     * Get the value of categorievoiture
     *
     * @return  string
     */ 
    public function getCategorievoiture()
    {
        return $this->categorievoiture;
    }

    /**
     * Set the value of categorievoiture
     *
     * @param  string  $categorievoiture
     *
     * @return  self
     */ 
    public function setCategorievoiture(string $categorievoiture)
    {
        $this->categorievoiture = $categorievoiture;

        return $this;
    }
}
